tests/Directory.Build.props

```diff
@@ -45,13 +46,14 @@
     <TargetingPackPath Condition="'$(BaseTargetingPackPath)' == ''">$(RootBinDir)TargetingPack\</TargetingPackPath>
   </PropertyGroup>
 
-  <!-- Provides properties for dependency versions and configures dependency verification/auto-upgrade. -->
-  <Import Project="$(ProjectDir)..\dependencies.props" />
-
   <!-- Add test-specific package restore sources. -->
   <PropertyGroup>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json;
+      https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
```

Comment:
Why were these added, and would it be possible to put them in NuGet.Config instead? Arcade is recommending to move away from `RestoreSources`: https://github.com/dotnet/arcade/blob/f8fc65bce78b79d00c66cb22ff658fb3a3e46dfb/Documentation/RestoreSourcesUpdateStatus.md.

Reply:
Well, we weren't using nuget.config in our test builds at all. But the move to unify the properties resulted in an empty RestoreSources at this point in the build. This then added a single entry, and that wasn't sufficient for restore to work. I've moved the test build process to use nuget.config and that appears to work fine. Nice catch.